js/components/volumeControls.js

```diff
@@ -184,8 +192,7 @@ VolumeControls.propTypes = {
       isMobile: React.PropTypes.bool.isRequired,
       volumeState: React.PropTypes.shape({
         volumeSliderVisible: React.PropTypes.bool.isRequired,
-        volume: React.PropTypes.number.isRequired,
-        muted: React.PropTypes.bool.isRequired
```

Comment:
I think this `muted` prop is still being used above

Reply:
Done.